test/blackbox-tests/test-cases/generate-sources.t

```diff
@@ -12,9 +12,10 @@ Generate the source of an executable in a subdir:
   > EOF
 
   $ dune exec --display short ./bar.exe
-        ocamlc .bar.eobjs/byte/dune__exe__Bar.{cmi,cmti}
-  File "dune", line 4, characters 0-23:
-  4 | (executable (name bar))
-      ^^^^^^^^^^^^^^^^^^^^^^^
-  Error: No rule found for bar.ml
```

Comment:
Again, it's unclear if we are accidentally making this test less useful by triggering the new error.

Reply:
I managed to restore this one.